ext/dom/tests/gh14652.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+GH-14652 segfault on object cloning
+--EXTENSIONS--
+dom
+--CREDITS--
+YuanchengJiang
+--FILE--
+<?php
+function varToString($var) {
+}
+$attr = new DOMAttr('category', 'books');
+$script1_dataflow = $attr;
+class test {
+private function __clone() {
+}
+}
+$clone = clone $script1_dataflow;
+echo "OK";
```

Comment:
Please make sure to close tests with `?>` so that phpt files can be run directly.